DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/mstpctl/linux/linux_mstpctl_impl.py

```diff
@@ -0,0 +1,116 @@
+from dent_os_testbed.lib.mstpctl.linux.linux_mstpctl import LinuxMstpctl
+
+
+class LinuxMstpctlImpl(LinuxMstpctl):
+    """
+        MSTPCTL is used for configuring STP parameters on bridges which have user-space
+        STP enabled. Currently, STP is disabled by default on the bridge. To
+        enable user-space STP, configure "brctl stp <bridge> on" or
+        "ip link set <bridge> type bridge stp_state 1" while ensuring that
+        /sbin/bridge-stp kernel helper script will return success (0) for
+        this operation.
+
+    """
+
+    def format_set(self, command, *argv, **kwarg):
+        params = kwarg['params']
+        cmd = 'mstpctl {} '.format(command)
+        if 'options' in params.keys():
+            cmd = 'mstpctl {}{} '.format(command, params['options'])
+        if 'bridge' in params:
+            cmd += '{} '.format(params['bridge'])
+        if 'revision' in params:
+            cmd += '{} '.format(params['revision'])
+        if 'name' in params:
+            cmd += '{} '.format(params['name'])
+        if 'max_age' in params:
+            cmd += '{} '.format(params['max_age'])
+        if 'fwd_delay' in params:
+            cmd += '{} '.format(params['fwd_delay'])
+        if 'max_hops' in params:
+            cmd += '{} '.format(params['max_hops'])
+        if 'hello_time' in params:
+            cmd += '{} '.format(params['hello_time'])
+        if 'ageing_time' in params:
+            cmd += '{} '.format(params['ageing_time'])
+        if 'stp_type' in params:
+            cmd += '{} '.format(params['stp_type'])
+        if 'enable' in params:
+            cmd += '{} '.format(params['enable'])
+        if 'priority' in params:
+            cmd += '{} '.format(params['priority'])
+        if 'tx_hold_count' in params:
+            cmd += '{} '.format(params['tx_hold_count'])
+        if 'cost' in params:
+            cmd += '{} '.format(params['cost'])
```

Comment:
This is good. Now you can reduce all of the duplicate code by using a loop:
```python
for key in ['bridge', 'port', 'name', 'ageing_time', ...]:
    if key in params:
        cmd += f'{params[key]} '
```
Also, don't forget about some special cases (on/off/auto)